src/analysis/normed_space/basic.lean

```diff
@@ -806,6 +806,17 @@ end prio
   [h : normed_algebra 𝕜 𝕜'] (x : 𝕜) : ∥algebra_map 𝕜 𝕜' x∥ = ∥x∥ :=
 normed_algebra.norm_algebra_map_eq _
 
+@[priority 100]
+instance to_normed_space {𝕜 : Type*} (𝕜' : Type*) [normed_field 𝕜] [normed_ring 𝕜']
```

Comment:
Both fields should be explicit arguments.

Reply:
Fixed in 29e139f